onnxruntime/core/providers/openvino/backends/basic_backend.cc

```diff
@@ -82,9 +82,7 @@ BasicBackend::BasicBackend(const ONNX_NAMESPACE::ModelProto& model_proto,
             ie_cnn_network_, hw_target, device_config, subgraph_context_.subgraph_name);
       }
 #else  // !IO_BUFFER_ENABLED
-      std::string prec_str = (global_context_.precision_str != "ACCURACY") ?
-                              global_context_.precision_str :
-                              global_context_.model_precision;
+      std::string prec_str = (global_context_.precision_str != "ACCURACY") ? global_context_.precision_str : global_context_.model_precision;
```

Comment:
Recommendation 
```suggestion
      std::string precision_string = (global_context_.precision_str != "ACCURACY") ? 
      global_context_.precision_str : global_context_.model_precision;
```

Reply:
Fixed the spacing. I think prec_str variable name is still readable.